test/Where.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai";
+import { argv0 } from "process";
+import { DangerLinq } from "../src";
+import { DangerLinqFunction } from "../src/DangerLinqFunction";
+
+describe("DangerLinqFunction.Map", () => {
+    it("Should be a function", async () => {
+      const Where = await DangerLinq(DangerLinqFunction.Map);
```

Comment:
To be consistent with the other test files, please use a lowercase `where` for the variable name.